### PR TITLE
Add napari-organoid-counter recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

--- 

@jaimergp, @christinab12, do you agree with being added as maintainers?

**Comment by christinab12** [comment]
@goanpeca yes thanks!

**Review by christinab12 — APPROVED** [review]

**Comment by goanpeca** [comment]
```
+ pip check
napari-aicsimageio 0.7.1 requires aicspylibczi, which is not installed.
napari-organoid-counter 0.1.0 has requirement napari[all]==0.4.15, but you have napari 0.4.16.
napari-organoid-counter 0.1.0 has requirement napari-aicsimageio==0.6.1, but you have napari-aicsimageio 0.7.1.
```

**Comment by christinab12** [comment]
@goanpeca sorry do you mind explaining where do I have napari 0.4.16 and napari-aicsimageio 0.7.1.? I think I have set it everywhere to napari[all]==0.4.15 and napari-aicsimageio==0.6.1

**Comment by goanpeca** [comment]
@christinab12 is there any reason to have the tight constraints to napari ? could they be relaxed >=  or <=0.5 or similar??

**Comment by christinab12** [comment]
I have just tested it with setting 
`install_requires =
    napari[all]>=0.4.15
    napari-aicsimageio>=0.6.1
`
in setup.cfg and everything seems to work also for 
`napari==0.4.16
napari-aicsimageio==0.7.2`

**Comment by goanpeca** [comment]
Still working on this!

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-organoid-counter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - napari_organoid_counter
+  commands:
+    - pip list
+    # - pip check  # Fails due to aicsimageio deps and fixed constraintes that were relaxed
+  requires:
+    - pip
+    - pyqt
```
Comment (xhochy) [inline recipes/napari-organoid-counter/meta.yaml:35]: Why is this listed here and not in run?

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/napari-organoid-counter/meta.yaml (hunk 1 above)
Comment (goanpeca) [reply recipes/napari-organoid-counter/meta.yaml:35]: @xhochy  napari no longer enforces a qt bidning by default, so when doing

`conda install napari -c conda-forge` or `pip install napari` you will still need to do

`install pyqt` or `install pyside` 


See: https://napari.org/stable/tutorials/fundamentals/installation.html


Screenshot 2023-10-23 at 5 35 20 PM

Comment (xhochy) [reply recipes/napari-organoid-counter/meta.yaml:35]: Ah! The best way forward would be to have a metapackage napari-qt (or a more general name without using napari) which builds two variants, one for each qt binding and let all napari recipes depend on this metapackage if they want qt.

Comment (goanpeca) [reply recipes/napari-organoid-counter/meta.yaml:35]: This was already discussed and voted against. 

See:
- https://github.com/conda-forge/napari-feedstock/pull/48
- https://github.com/conda-forge/napari-feedstock/issues/47

The napari team is ok with having this as is for both conda and pip.

Comment (xhochy) [reply recipes/napari-organoid-counter/meta.yaml:35]: Not really what I think is best, but also not an area where I have something at stake, so let's merge this.

**Review by xhochy — APPROVED** [review]

**Pull request merged**